tests/test_transformation/test_arcsin_transformer.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.transformation import ArcsinTransformer
+
```

Comment:
Hi @tomtom-95 

These tests are really great, they test overall the error catching bits of the transformer. But we still need tests to corroborate the output of the arcsin(sqrt(x)) transformation and then the inverse transformation to recover the original values.

We would need to include a test like this one:

https://github.com/feature-engine/feature_engine/blob/248465e111416f88a4784b5e2e13a83d47e1f00e/tests/test_transformation/test_reciprocal_transformer.py#L8-L34